copernicusmarine/core_functions/models.py

```diff
@@ -46,12 +49,24 @@ class StatusMessage(str, Enum):
 
     SUCCESS = "The request was successful."
     DRY_RUN = "The request was successful but no data was transferred."
-    ERROR = "An error occurred during the request."
-    NO_DATA_TO_DOWNLOAD = "No data to download from the remote server corresponding to your request."  # noqa: E501
     FILE_LIST_CREATED = "The request created a file list and then stopped."
+    NO_DATA_DOWNLOADED = "No data was downloaded."
+    DATA_OVERWRITTEN = "The request overwrited the data."
+    NO_DATA_TO_DOWNLOAD = "No data to download from the remote server corresponding to your request."  # noqa: E501
+    ERROR = "An error occurred during the request."
+
+
+class StatusFile(str, Enum):
+    """
+    Status of the files in the get
+    """
+
+    DOWNLOAD = "DOWNLOAD"
+    IGNORED = "IGNORED"
```

Comment:
```suggestion  
    DOWNLOADED = "DOWNLOADED"
    IGNORED = "IGNORED"
    OVERWRITTEN = "OVERWRITTEN"
```

And then add descriptions and add this to the documentation by putting this class in the `__init__` and the reponse type rst file

Reply:
Done!